src/modules/account/store/action.test.js

```diff
@@ -18,4 +19,14 @@ describe('actions: account', () => {
       expect(setCurrentAccount()).toEqual(expectedAction);
     });
   });
+
+  describe('add account', () => {
+    it('should create an action to add account', () => {
+      const expectedAction = {
+        type: actionTypes.addAccount,
+      };
+
+      expect(addAccount()).toEqual(expectedAction);
+    });
+  });
```

Comment:
@soroushm `deleteAccount` is not tested here

Reply:
done